### PR TITLE
Add LAO

### PR DESCRIPTION
**Pull request by Ro5s**
#TheLAO

### CODE REVIEW

File: src/namespaces.json [hunk 1]
```diff
@@ -231,5 +231,17 @@
     "min": 0,
     "invalid": [],
     "visible": false
+  },
+  "lao": {
+    "key": "lao",
+    "symbol": "LAO",
+    "name": "LAO",
```
Comment (bonustrack) [inline src/namespaces.json:238]: I'd suggest to put the full name here
```suggestion
    "name": "The LAO",
```

File: src/assets/skins/lao.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.lao {
```
Comment (bonustrack) [inline src/assets/skins/lao.scss:1]: The skin file should be linked here: https://github.com/balancer-labs/snapshot/blob/develop/src/assets/skins/index.scss

Comment (Ro5s) [reply src/assets/skins/lao.scss:1]: ah, good point. will stitch in~~

Comment (Ro5s) [reply src/assets/skins/lao.scss:1]: ok, I think I was able to resolve :+1: @bonustrack thanks for quick lookover!

**Review by bonustrack — APPROVED** [review]
Looking great!

**Pull request merged**